owncloudApp/src/main/java/com/owncloud/android/presentation/previews/PreviewTextViewModel.kt

```diff
@@ -66,6 +69,15 @@ class PreviewTextViewModel(
                 remove(FileMenuOption.MOVE)
                 remove(FileMenuOption.COPY)
                 remove(FileMenuOption.SYNC)
+
+               if(isTypeMarkdown){
+                   if(isAscii){
```

Comment:
KtLint would complain as well about not separating the `if` from the condition and the `{`
```suggestion
               if (isTypeMarkdown) {
                   if (isAscii) {
```
Android Studio can make this tiny changes automatically for you in Code > Reformat code 👍